bin/v-backup-user

```diff
@@ -458,7 +463,7 @@ if [ -n "$MAIL_SYSTEM" ] && [ "$MAIL" != '*' ]; then
 	# Parsing domain exclusions
 	conf="$USER_DATA/mail.conf"
 	for domain in $(search_objects 'mail' 'SUSPENDED' "*" 'DOMAIN'); do
-		check_exl=$(echo "$MAIL" | tr ',' '\n' | grep "^$domain$")
+		check_exl=$(echo "$MAIL" | tr ',' '\n' | grep "^$domain\|\*$")
```

Comment:
Issue is caused by:

if * is used is should always be blocked

reverting to original should work fine

Reply:
Not sure what you mean? If the exclusion is * (on its own), then this should catch it?

`if [ -n "$MAIL_SYSTEM" ] && [ "$MAIL" != '*' ]; then`

Reply:
Yeah.  But some reason it doesn't work either with multiple domains present ...

I think we should revert the mail domain changes and only do it for the web domains for now...

Reply:
Sure - I was only trying to tweak it so the other bits worked as well :) (I don't use the other backup exclusions). I've created the "fixed" version just for web and submitted it

Reply:
I don't think the need is there as much for mail then for web to globally block certain accounts from backed up

For web it makes a lot more sense imo